indykite_sdk/authorization/is_authorized.py

```diff
@@ -81,5 +86,19 @@ def is_authorized_property_filter(self, type_filter, value, resources=[], action
 def request_resource(resources):
     res = []
     for r in resources:
-        res.append(pb2.IsAuthorizedRequest.Resource(id=r.id, label=r.label))
+        actions = []
+        for a in r.actions:
+            actions.append(a)
+        res.append(pb2.IsAuthorizedRequest.Resource(id=r.id, type=r.type, actions=actions))
     return res
```

Comment:
This should be equivalent and easier to read.
```suggestion
    def request_resource(resources):
    return [
        pb2.IsAuthorizedRequest.Resource(id=r.id, type=r.type, actions=list(r.actions))
        for r in resources
    ]
```

Reply:
thank you for the improvement